index.html

```diff
@@ -104,6 +105,7 @@ <h2 class="main-heading">I'm Shaili<br />Glad to see you!</h2>
       </section>
       <!-- #### Project-Work-section #### -->
       <section class="projects" id="portfolio">
```

Comment:
- [ ] As suggested by the previous code reviewer and as written in the project requirements. I kindly suggest that you make sure the projects section is created dynamically with the information from an array of objects.

## EXAMPLE OF PROJECTS ARRAY OF OBJECTS
```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```

- [ ] Kindly also ensure Modal popup data is populated with the data from the projects array also.